src/manifest.json

```diff
@@ -35,7 +35,6 @@
       "captivePortal",
       "identity",
       "idle",
-      "management",
```

Comment:
LOL, I was curious about this so I checked [the docs](https://www.devdoc.net/web/developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/management/uninstallSelf.html):
> This API does not require the "management" [API permission](https://www.devdoc.net/web/developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions.html).

&hellip; but then realized there was a lengthy discussion in The Jira ticket about this.